src/timew.h

```diff
@@ -57,6 +54,11 @@ std::vector <Range>     getUntracked      (Database&, const Rules&, Interval&);
 Interval                getLatestInterval (Database&);
 Range                   getFullDay        (const Datetime&);
 
+template <typename It>
+std::pair <It, It>      intersects        (It begin, It end, const Range& day_range);
+
+#include <data.inl>
```

Comment:
Why not include the template code here? Just like in `libshared/src/format.h` for example

Reply:
I thought that your preference wasn't to add implementations in the .h files, but I can move it when this is updated.